tests/jax/systems/mappo_system_test.py

```diff
@@ -54,36 +78,37 @@ def test_except_trainer(
 
     # Optimizer.
     optimizer = optax.chain(
-        optax.clip_by_global_norm(40.0), optax.scale_by_adam(), optax.scale(-1e-4)
+        optax.clip_by_global_norm(40.0),
+        optax.adam(1e-4),
     )
 
     # Build the system
     test_full_system.build(
         environment_factory=environment_factory,
         network_factory=network_factory,
         logger_factory=logger_factory,
-        checkpoint_subpath=checkpoint_subpath,
+        experiment_path=checkpoint_subpath,
         optimizer=optimizer,
-        executor_parameter_update_period=20,
-        multi_process=False,
+        executor_parameter_update_period=1,
+        multi_process=True,
         run_evaluator=True,
         num_executors=1,
+        max_queue_size=500,
         use_next_extras=False,
-        sample_batch_size=2,
+        sample_batch_size=5,
+        checkpoint=True,
+        nodes_on_gpu=[],
+        lp_launch_type=lp.LaunchType.TEST_MULTI_THREADING,
```

Comment:
based on the fixes in the other integration tests, is it not possible to have this as MULTI_PROCESSING @OmaymaMahjoub ?

Reply:
I am currently investigating it

Reply:
Based on the chat with the other team members, I found out that it is challenging at the current moment to test the system using `lp_launch_type=lp.LaunchType.TEST_MULTI_PROCESSING` so we will just have the tests in Multi_threading case :sweat: at the current time

Reply:
I see that in the other integration tests, the single process test has MULTI_PROCESSING launch type and multi_process has MULTI_THREADING so yes, I agree that it is fair to merge this: it is a big step to be able to run in multi_process mode now :smile:. Ideally, we should not have to copy the node_fn `(node_fn = copy_node_fn(node_fn))` as it means that multi_threading launch type is shared. Happy to close this issue but please first create an issue to investigate launch_type.

Reply:
Agreed :slightly_smiling_face: After we create an issue we could merge this.

Reply:
Does the issue will be closed with this PR or We will create an issue for the future?

Reply:
I have created the issue you can find it here  #690